lookout/style/typos/analyzer.py

```diff
@@ -13,16 +16,17 @@
 import pandas
 from sourced.ml.algorithms import TokenParser, uast2sequence
 
-from lookout.style.common import merge_dicts
+from lookout.style.common import load_jinja2_template, merge_dicts
 from lookout.style.format.utils import generate_comment
 from lookout.style.typos.corrector_manager import TyposCorrectorManager
-from lookout.style.typos.utils import Candidate, Columns, flatten_df_by_column
+from lookout.style.typos.utils import Candidate, Columns, flatten_df_by_column, TEMPLATE_DIR
 
 TypoFix = NamedTuple("LineFix", (
     ("head_file", File),  # file from head revision
     ("line_number", int),  # line number for the comment
-    ("token", str),  # token where typo is found
-    ("candidates", List[Candidate]),  # Candidates for token fix
+    ("identifier", str),  # identifier where typo is found
+    ("candidates", Mapping[str, Iterable[Candidate]]),  # Candidates for token fix
+    ("identifiers", Iterable[Candidate]),  # Suggested identifiers
```

Comment:
Also `Sequence`

Reply:
Also, It is a little bit confusing to have `identifier` and `identifiers`, don't you think?
Maybe, let's rename `candidates ` to `token_candidates` and `identifiers` to `identifier_candidates`?

Reply:
`identifier_candidates` :+1: 
`Sequence` :-1: